Specs/DateTime/French/PercentModelPercentMode.json

```diff
@@ -0,0 +1,157 @@
+[
+    {
+        "Input": "100%",

```

Comment:
Please focus on the files that end in Extractor, Parser, and Model. The only ones extra at first should be the ones with ExperimentalMode. All other modes and extra suffixes can wait as they are lower priority.